src/pages/EditMonitor.vue

```diff
@@ -183,7 +183,7 @@
                             <!-- Interval -->
                             <div class="my-3">
                                 <label for="interval" class="form-label">{{ $t("Heartbeat Interval") }} ({{ $t("checkEverySecond", [ monitor.interval ]) }})</label>
-                                <input id="interval" v-model="monitor.interval" type="number" class="form-control" required min="20" step="1">
+                                <input id="interval" v-model="monitor.interval" type="number" class="form-control" required min="1" step="1">
```

Comment:
This has already been implemented by #1740. #1740 also added a warning for the risk of poor performance if it was reduced below 20